test/nn/models/test_hetero_explainer.py

```diff
@@ -45,14 +85,18 @@ def forward(self, x, edge_index):
         return self.conv2(x, edge_index)
 
 
-def test_set_clear_mask():
-    data = HeteroData()
-    data['paper'].x = torch.randn(50, 16)
-    data['author'].x = torch.randn(30, 8)
-    data['paper', 'paper'].edge_index = get_edge_index(50, 50, 200)
-    data['author', 'paper'].edge_index = get_edge_index(30, 50, 100)
-    data['paper', 'author'].edge_index = get_edge_index(50, 30, 100)
+class HeteroSage(torch.nn.Module):
+    def __init__(self, metadata: Metadata):
+        super().__init__()
+        self.graph_sage = to_hetero(GraphSAGE(), metadata, debug=False)
 
+    def forward(self, x_dict, edge_index_dict, *args) -> torch.Tensor:
+        assert args
```

Comment:
Not really sure what we test here. Can you clarify?

Reply:
Added a comment. We are checking if an additional argument gets passed.